### PR TITLE
llm tests and uml, tts

### PR DESCRIPTION
**Pull request by vladutdinu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,10 @@
 	"dependencies": {
 		"@supabase/ssr": "^0.5.1",
 		"@supabase/supabase-js": "^2.45.4",
-		"marked": "^14.1.2"
+		"@viz-js/viz": "^3.9.0",
```
Comment (smetytech) [inline package.json:44]: Please remove

File: src/lib/scripts/llm.script.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { PRIVATE_GROQ_API_KEY } from '$env/static/private';
+import Groq from 'groq-sdk';
+
+const MODEL = 'llama-3.1-70b-versatile';
+const TEMPERATURE = 0;
+
+const groq = new Groq({ apiKey: PRIVATE_GROQ_API_KEY });
```
Comment (smetytech) [inline src/lib/scripts/llm.script.ts:7]: Please move this bellow the UPPERCASE constants

File: src/lib/scripts/speak.script.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export function speak(message: string) {
```
Comment (smetytech) [inline src/lib/scripts/speak.script.ts:1]: You can name this `textToSpeach` and the param as `text`

File: src/routes/api/llm/uml/+server.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import { generateUML } from 'src/lib/scripts/llm.script';
+import { createRequire } from 'module';
+const require = createRequire(import.meta.url);
+const plantumlEncoder = require('plantuml-encoder');
+
+// API endpoint handler
+export const POST: RequestHandler = async ({ locals: { supabase }, request }) => {
+    const { content } = await request.json();
+    const createdTTS = await generateUML(content);
+    return new Response('http://www.plantuml.com/plantuml/img/' + plantumlEncoder.encode(createdTTS), { headers: { 'Content-Type': 'text/plain' } });
```
Comment (smetytech) [inline src/routes/api/llm/uml/+server.ts:11]: Please extract the url in a separate variable

**Review by smetytech — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (vladutdinu) [reply package.json:44]: gata

File: src/lib/scripts/llm.script.ts (hunk 2 above)
Comment (vladutdinu) [reply src/lib/scripts/llm.script.ts:7]: gata

File: src/lib/scripts/speak.script.ts (hunk 3 above)
Comment (vladutdinu) [reply src/lib/scripts/speak.script.ts:1]: gata

Comment (vladutdinu) [reply src/lib/scripts/speak.script.ts:1]: Speech*

File: src/routes/api/llm/uml/+server.ts (hunk 4 above)
Comment (vladutdinu) [reply src/routes/api/llm/uml/+server.ts:11]: data

**Review by smetytech — COMMENTED** [review]
Nice

**Pull request merged**